txscript/opcode.go

```diff
@@ -2899,6 +2900,51 @@ func opcodeCheckSigAltVerify(op *opcode, data []byte, vm *Engine) error {
 	return err
 }
 
+// opcodeTAdd adds value to the treasury balance.
```

Comment:
This is not really true.  While it does indeed ultimately do that, it doesn't happen here and thus this is misleading.